.github/scripts/get_workflow_job_id.py

```diff
@@ -6,6 +6,17 @@
 import os
 import argparse
 
+
+def handle_bad_status(response):
+    if response.status_code != 200:
+        exception_message = (
+            "Is github alright?",
+            f"Recieved status code '{response.status_code}' when attempting to retrieve runs:\n",
+            f"{response.content}"
+        )
+        raise Exception(exception_message)
```

Comment:
Why basic exception?
```suggestion
        raise RuntimeError(exception_message)
```